### DISCUSSION TITLE
Reandering issues

### DISCUSSION THREAD

**Discussion by thedynomo** [community/community#142685] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Sometimes when I host my static website on github the format changes or scatters. I really don't know, I've tried everything but nothing. Any similar experiences?

**Comment by justdeen** [comment]
I think I know the issue. Do you use pixels for positioning elements? 
If yes then don't use pixels for positioning instead use rems. Your desktop browser renders it differently from your mobile browser

**Comment by Imran-2022** [comment]
This often happens due to incorrect file paths or case sensitivity issues. Make sure all CSS/JS links are correctly referenced and match the filenames exactly.

**Reply by thedynomo** [reply]
Okay I'll check it out. Thanks!!

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142683#discussioncomment-11078623) for reference to why this discussion was closed and locked. Thanks!
